Update README.pt-br.md

viniciusifsp opened the pull request:
Proposta de atualização da tradução para pt-BR.

github-education reviewed (changes requested):
**Hi viniciusifsp,**
**Welcome to graduation!**


### I have a few items I need you to take care of before I can merge this PR:

* *Uh Oh! I've found some issues with where you have created your files!* 
	*The required markdown file does not exist, please ensure the file `_data/viniciusifsp/viniciusifsp.md` exists*
*Please ensure all changes are contained within the `_data/viniciusifsp/` directory. Invalid file paths:* 

	* `translations/README.pt-br.md`


Feel free to re-request a review from me and I'll come back and take a look!

viniciusifsp commented:
Proposal to update the translation to pt-BR:

- Removed extra spaces
- Possible more accurate translation
- Added some parts that are suppressed from the original English version
